Dockerfile.agnos

```diff
@@ -69,34 +69,36 @@ COPY --from=agnos-compiler-capnp /tmp/capnproto.deb /tmp/capnproto.deb
 RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./capnproto.deb
 
 # Install openpilot dependencies
-COPY ./userspace/openpilot_dependencies.sh /tmp/agnos/
-RUN /tmp/agnos/openpilot_dependencies.sh
 COPY ./userspace/openpilot_python_dependencies.sh /tmp/agnos/
 RUN /tmp/agnos/openpilot_python_dependencies.sh
 
 # Use other pre-compiled packages
 COPY --from=agnos-compiler-ffmpeg /tmp/ffmpeg.deb /tmp/ffmpeg.deb
 RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./ffmpeg.deb
 
-RUN export PATH="/usr/local/pyenv/bin:/usr/local/pyenv/shims:$PATH" && \
-    export PYENV_ROOT="/usr/local/pyenv" && \
-    eval "$(pyenv init -)" && \
-    eval "$(pyenv virtualenv-init -)" && \
-    export MAKEFLAGS="-j$(nproc)" && \
-    pip install pyqt5-sip==12.12.1 && \
-    pip install pyqt5==5.15.9 --verbose --config-settings --confirm-license= && \
-    pyenv rehash
+ENV XDG_DATA_HOME="/usr/local"
+ENV PATH="$XDG_DATA_HOME/.cargo/bin:$PATH"
+
+COPY ./userspace/files/PyQt5-5.15.9-cp38-abi3-manylinux_2_17_aarch64.whl /tmp/agnos/
```

Comment:
From this [comment](https://github.com/commaai/agnos-builder/pull/278#issuecomment-2264599903) the way to go is to still build PyQt5 for now but make sure its somewhere in the build where other changes to the dockerfile dont break the cache. Also its unrelated to the uv introduction, so could we do this in another PR and remove it here?

Reply:
according to docker build cache invalidation general rules,
"During cache lookup, cache is invalidated if the file metadata has changed for any of the files involved."
the docker cache should not bust as long as the wheel file metadata is not modified. I have already built the wheel and setup in dockerfile so lets keep using the pre-built PyQt5 wheel also the build time optimization is much needed.

Reply:
according to docker build cache invalidation general rules,
"During cache lookup, cache is invalidated if the file metadata has changed for any of the files involved."
the docker cache should not bust as long as the wheel file metadata is not modified. I have already built the wheel and setup in dockerfile and using the pre-built PyQt5 wheel also provides the much needed build time optimization. Unless updating to pyqt6, I suggest to keep using the pre-built wheel.